features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/constants.jag

```diff
@@ -18,7 +18,7 @@
  */
 
 var MGT_TRANSPORT = "https://";
-var SETTINGS_REST_API_URL_SUFFIX = "/api/am/store/v1.0/settings";
+var SETTINGS_REST_API_URL_SUFFIX = "/api/am/store/v1.1/settings";
```

Comment:
Shouldn't we make this v1 instead?

Reply:
This suffix used in UI level. I think we need keep this to latest version.